ClassesHomework/Cat.cs

```diff
@@ -6,10 +6,59 @@
 
 namespace ClassesHomework
 {
-    // Write your Cat class here
-    //class Cat
-    //{
-        
-    //}
+    class Cat : Animal
+    {
+        public Cat()
+        {
+            IsSitting = false;
+            Age = 0;
+
+        }
+
+        public string Breed { get; private set; }
```

Comment:
You're getting a warning here because the Breed property is "_hiding_" Animal.Breed.

If you intend to just use the parent class's property (and not give it a different definition in the child class), then you don't need it to appear at all in the child class.

If you intend to **override** the parent class's property (give a different definition in the base class), then there are two keywords that need to be used. In the parent class, tag the property as `virtual`. You might be able to guess the keyword that goes in the child class, or the warning (after you've added `virtual` in the parent) will give a clue.

This all goes for methods as well as properties.